[#17] 친구추가 요청 거절 기능 구현

msugo1 opened the pull request:
## FriendController

### `rejectFriendRequest` 메소드 추가

- 친구추가 거절에 대한 요청을 받을 컨트롤러 메소드

## FriendService

### `rejectFriendRequest` 메소드 추가

- 친구추가 거절 요청에 대한 비즈니스 로직을 수행할 메소드

- DB에 등록된 상태가 `친구 요청을 받은 상태(RECEIVED)`와 일치하는 경우에만 요청을 Mapper로 전달

## 테스트

- 친구추가 요청 거절에 대한 로직을 테스트 할 테스트 작성

- 포스트맨을 이용한 테스트

Commit:
[#17] 친구추가 요청 거절 기능 구현

FriendController

* rejectFriendRequest 메소드 추가

- 친구추가 거절에 대한 요청을 받을 컨트롤러 메소드

FriendService

* rejectFriendRequest 메소드 추가

- 친구추가 거절 요청에 대한 비즈니스 로직을 수행할 메소드

: DB에 등록된 상태가 `친구 요청을 받은 상태(RECEIVED)`와 일치하는
경우에만 요청을 Mapper로 전달

테스트

- 친구추가 요청 거절에 대한 로직을 테스트 할 테스트 작성

- 포스트맨을 이용한 테스트

f-lab-dev reviewed (approved):
테스트케이스가 꼼꼼해서 좋네요~ `@ResponseStatus` 부분만 고쳐주시면 될 것 같네요~

f-lab-dev commented:
그리고 친구 관련 테이블들 sql파일도 만들어서 올려주시면 좋을 것 같아요

Commit:
[#17] 피드백 반영

- 불필요한 `@ResponseStatus` 제거 (HttpStatus.OK)
- `Friends` 테이블의 Schema를 담은 `friends.sql` 파일 추가
